apps/ledger-live-mobile/src/newArch/features/Accounts/__integrations__/addAccount.integration.test.tsx

```diff
@@ -13,7 +13,19 @@ describe("AddAccount", () => {
         settings: {
           ...state.settings,
           readOnlyModeEnabled: false,
-          overriddenFeatureFlags: { llmWalletSync: { enabled: true } },
+          overriddenFeatureFlags: {
+            llmWalletSync: {
+              enabled: true,
+              params: {
+                environment: "STAGING",
+                watchConfig: {
+                  pollingInterval: 10000,
+                  initialTimeout: 5000,
+                  userIntentDebounce: 1000,
```

Comment:
can the default be just `{}` ? these are all falsy properties in `watchConfig`

so we can be more DRY (don't repeat yourself) since this value is currently present 3 times in the codebase (in the implem + in LLM and LLD config here)